### DISCUSSION TITLE
transitionSpec in iOS

### DISCUSSION THREAD

**Discussion by Razorholt** [software-mansion/react-native-screens#1171] [Q&A] [answered]
Can I use transitionSpec in iOS to change the transition speed of a stackPresentation?

Thanks,
Razor

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @Razorholt,
unfortunately, as of yet, there is no way of changing transition speed within the native stack from JavaScript. If you have to change the animation speed you'll need to get into the native code and use `patch-package`.

https://github.com/software-mansion/react-native-screens/blob/66b1d974dcefe9b21ef30c68627dd98296ea4bfe/ios/RNSScreenStackAnimator.m#L14

Cheers

**Reply by Razorholt** [reply]
Thank you!

**Reply by hirbod** [reply]
Isn't this something that could be configured and used on buildtime? Specially for expo folks without custom dev clients or bare workflows, patching is impossible @kacperkapusciak. There must be a way to pass the value.

**Reply by WoLewicki** [reply]
@Hirbod can you check if #1259 is what you need?

**Reply by hirbod** [reply]
That's awesome, thanks @WoLewicki

**Reply by WoLewicki** [reply]
What do you think of adding this option on Android? It can be a little trickier, as I mentioned in PR's description.

**Reply by hirbod** [reply]
It would be lovely as well. I think that feature parity is always a good thing. The most awesome thing would be gestures on android, but that's another story.
